oligo_designer_toolsuite/oligo_specificity_filter/_filter_bowtie2.py

```diff
@@ -0,0 +1,203 @@
+import os
+import re
+from pathlib import Path
+from subprocess import Popen
+
+import pandas as pd
+from joblib import Parallel, delayed
+
+from . import SpecificityFilterBase
+
+
+class Bowtie2(SpecificityFilterBase):
+    """This class filters probes based on the Bowtie 2 read alignment tool. It is recommended to use Bowtie 2 instead of Bowtie for reads longer than about 50 bp, as it gives better performance.
```

Comment:
The Bowtie 2 filter gives **an alignment** score to each ... the more **similar**...
 an **alignment** score greater than min_score...
    :param min_score: User defined **threshold** for **alignment** score. If specified, the Bowtie 2 filter filters out probes with an **alignment** score...